requirements/base.txt

```diff
@@ -5,8 +5,8 @@
 # edX Internal Requirements
 git+https://github.com/edx/XBlock.git@xblock-0.4.1#egg=XBlock==0.4.1
 
-# We are now forking the edx-submissions repo for ora score override function.
-git+https://github.com/Stanford-Online/edx-submissions.git@77b5171791c34bc884cf94db1c602dd1b7f587d7#egg=edx-submissions
+# edx-submissions
+git+https://github.com/edx/edx-submissions.git@0.1.3#egg=edx-submissions==0.1.3
```

Comment:
Well this is just the version that matches our current ORA2 merge point, with the new merge this will get updated to 1.1.4